sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -97,7 +97,7 @@ export function nodeConfig({ test = false, production = false } = {}) {
   return baseConfig;
 }
 
-export function browserConfig({ test = false, production = false } = {}) {
+export function browserConfig(test = false) {
```

Comment:
Service Bus has been shipping the browser bundle as part of the package.
Am guessing there needs to be a change to the package.json file to remove it.

We can do that and the rollup change for service bus in a separate PR. Please log a separate issue for it.

Reply:
> Service Bus has been shipping the browser bundle as part of the package.

I am not seeing a reference to `browser/service-bus.min.js`. I guess the bundle might have been removed in previous commits

Reply:
It has been shipping the unminified version `browser/service-bus.js` since the beginning so I think we are good here.